tests/conditional_formatting/conditional_formatting_panel_component.test.ts

```diff
@@ -440,6 +440,54 @@ describe("UI of conditional formats", () => {
       );
     });
 
+    test("Can remove a valid, invalid or empty range", async () => {
+      await click(fixture, selectors.buttonAdd);
+      await nextTick();
+
+      await setInputValueAndTrigger(selectors.ruleEditor.editor.operatorInput, "BeginsWith");
+      setInputValueAndTrigger(".o-selection-input input", "A1:A4");
+      expect(document.querySelectorAll(".o-selection-input input")).toHaveLength(1);
+
+      // Add & remove a valid range
+      await simulateClick(".o-add-selection");
+      const range1 = document.querySelectorAll(".o-selection-input input")[1];
+      await setInputValueAndTrigger(range1, "B1:B4");
+      const remove1 = document.querySelectorAll(".o-remove-selection")[1];
+      await simulateClick(remove1);
+      expect(document.querySelectorAll(".o-selection-input input")).toHaveLength(1);
+
+      // Add & remove an invalid range
+      await simulateClick(".o-add-selection");
+      const range2 = document.querySelectorAll(".o-selection-input input")[1];
+      await setInputValueAndTrigger(range2, "B1:MERA");
```

Comment:
a bit of nitpick but I'd rather have a stirng that explicitely says that the range is invalid like `invalid-range`  or `1:2:3` or even just `MERA` because the range you wrote here is invalid because it goes out out the bounds of the spreadsheet and not because it had a faulty syntax